packages/data-provider/src/utils/formatUtils.ts

```diff
@@ -62,3 +62,15 @@ export function isValidDateFormat(value: string): boolean {
 
 	return !isNaN(timestamp);
 }
+/**
+ * If the input is a string, it converts it into an array containing that string.
+ * Otherwise, it returns the input as-is.
+ * @param {string | string[] | undefined} value
+ * @returns {string[] | undefined}
+ */
+export function stringToArray(value: string | string[] | undefined): string[] | undefined {
+	if (typeof value === 'string') {
+		return [value];
+	}
+	return value;
+}
```

Comment:
This works and is a good solution. I have similar utility I use throughout lectern that I thought I'd share as a general solution: 

```ts
/**
 * Ensure a value is wrapped in an array.
 *
 * If passed an array, return it without change. If passed a single item, wrap it in an array.
 * @param val an item or array
 * @return an array
 */
export const asArray = <T>(val: T | T[]): T[] => (Array.isArray(val) ? val : [val]);
```

https://github.com/overture-stack/lectern/blob/develop/packages/dictionary/src/utils/typeUtils.ts#L20C1-L27C85

Reply:
Thanks for sharing this solution. Will use `asArray` function as I see it's more generic and will always return an array `(string | undefined)[]`

Reply:
While the implementation for both your functions is basically the same 👍 (a condition with the same two outcomes), the name for both of your functions would lead me to believe the functions do something entirely different. Naming things is hard, for sure... but hear me out:

- `String to Array` makes me think you'd parse `blah, blah, blah` into an array with the three "blah" strings.
- `asArray` evokes some sort of type assertion.
- ultimately neither name indicates "I'll wrap your value in an array, unless the value is already an array"

Not sure whether suggesting `arrayFromValue` or `valueToArray` makes it better, but perhaps addressing possible "expectations" from what the function may do (e.g. parsing a string into an array if possible, rather than just wrapping it) may help later.